packages/core/src/Parser.ts

```diff
@@ -132,6 +133,7 @@ export class Parser {
         // Start with the first child and the whole tree will be parsed.
         if (node.childNodes.length) {
             let currentContext: ParsingContext = {
+                parseNode: this.parseNode.bind(this, vDocumentMap),
```

Comment:
@Zinston This is how Christophe managed to pass the "generic" parse function to the others. When I left yesterday you were going to try something along the line of `this.editor.parse('dom', vnode)`, did it work ? (Well, technically it was for the renderer but you got my point.)